### ISSUE TITLE
support bundling with rollup

### ISSUE THREAD

**Issue by pravi** [wycats/handlebars.js#1279]
I'm trying to bundle handlebars with rollup (as webpack has a long list of dependencies), I get the following error. Can you support bundling with rollup?

```
$ rollup ./lib/handlebars.js 
'default' is not exported by lib/handlebars/compiler/parser.js (imported by lib/handlebars/compiler/base.js). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
Error: 'default' is not exported by lib/handlebars/compiler/parser.js (imported by lib/handlebars/compiler/base.js). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
    at Module.trace (/usr/lib/nodejs/rollup/src/Module.js:362:30)
    at Module.traceExport (/usr/lib/nodejs/rollup/src/Module.js:388:29)
    at Module.trace (/usr/lib/nodejs/rollup/src/Module.js:360:36)
    at ModuleScope.findDeclaration (/usr/lib/nodejs/rollup/src/ast/scopes/ModuleScope.js:47:22)
    at Scope.findDeclaration (/usr/lib/nodejs/rollup/src/ast/scopes/Scope.js:92:39)
    at Identifier.bind (/usr/lib/nodejs/rollup/src/ast/nodes/Identifier.js:7:29)
    at /usr/lib/nodejs/rollup/src/ast/Node.js:6:34
    at AssignmentExpression.eachChild (/usr/lib/nodejs/rollup/src/ast/Node.js:21:5)
    at AssignmentExpression.bind (/usr/lib/nodejs/rollup/src/ast/Node.js:6:8)
    at AssignmentExpression.bind (/usr/lib/nodejs/rollup/src/ast/nodes/AssignmentExpression.js:29:13)
Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki

```

**Comment by msroberts** [comment]
This is also an issue for me: because the compiled templates js file does not export anything, rollup does not modify the global Handlebars variable.

Related: https://github.com/wycats/handlebars.js/issues/1254, https://github.com/wycats/handlebars.js/issues/955

**Comment by nknapp** [comment]
Could you try manually changing the last lines of `lib/handlebars/compiler/parser.js` from 

```
exports.__esModule = true;
exports['default'] = handlebars;
```
to
```
export default handlebars;
```

This change would revert part of commit [efddc3](https://github.com/wycats/handlebars.js/commit/efddc3c09cacd6719a8206eeb3787a0c3aabb174#diff-de54cca7beb2a12881b485e404d4c845) and it does reduce code-coverage in the istanbul report. But this reduction seems to be a reporting issue. Maybe this can be handed differently.

At first, it would be interesting if the change solves your problem.

**Comment by nknapp** [comment]
This should be resolved with #1315
